lib/preprocessor.js

```diff
@@ -1,8 +1,7 @@
-var path = require('path')
 var fs = require('graceful-fs')
 var crypto = require('crypto')
 var mm = require('minimatch')
-var extensions = require('./binary-extensions.json').extensions
+var isBinaryFile = require('isBinaryFile')
```

Comment:
looks like this needs to be downcased to `require('isbinaryfile')` (see travis failures)